packages/server/src/database/service.ts

```diff
@@ -3,23 +3,17 @@ import knex, { Knex } from 'knex'
 import path from 'path'
 import { Service } from '../base/service'
 import { Table } from '../base/table'
-import { ConfigService } from '../config/service'
 import { Logger } from '../logger/types'
 
 export class DatabaseService extends Service {
   public knex!: Knex
   private url!: string
```

Comment:
```suggestion
  private url!: string | undefined
```

Then you can remove the exclamation point at the end of `process.env.DATABASE_URL` on line 16.

Reply:
I prefer having the !